descwl_shear_testing/coadd_obs.py

```diff
@@ -0,0 +1,99 @@
+"""
+A container for coadd observations.
+"""
+import galsim
+
+
+class CoaddObs(object):
+    """
+    Simple container for coadding SEObs and retrieving data.  It is assumed
+    that the psf is the same for all images, the wcs is desribed by a spatially
+    constant jacobian
+
+    Parameters
+    ----------
+    data: OrderedDict
+        keyed by band, with each element a list of SEObs
+    """
+    def __init__(self, data):
+        self._data = data
+        self._make_coadd()
+
+    @property
+    def image(self):
+        """
+        Get a reference to the image array
+        """
+        return self._image
+
+    @property
+    def weight(self):
+        """
+        Get a reference to the weight image array
+        """
+        return self._weight
+
+    @property
+    def jacobian(self):
+        """
+        get a reference to the jacobian
+        """
+        return self._jacobian
+
+    @property
+    def psf(self):
+        """
+        getter for psf observation
+        """
+        return self._psf
+
+    def _make_coadd(self):
```

Comment:
I think maybe this should be a function that returns another SEObs (or more general renamed class) with the coadded data.

Reply:
I used this because the metadetect code expects an observation